rust/relay/Cargo.toml

```diff
@@ -2,6 +2,16 @@
 name = "relay"
 version = "0.1.0"
 edition = "2021"
+license = "APL-2.0"
+authors = ["firezone developers"]
```

Comment:
@jamilbk Do we have a email address that we could put in here?

Reply:
Yep! `code-owners@`